hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -142,12 +142,8 @@ private Constants() {
   public static final String SERVER_SIDE_ENCRYPTION_ALGORITHM =
       "fs.s3a.server-side-encryption-algorithm";
 
-  /**
-   * The standard encryption algorithm AWS supports.
-   * Different implementations may support others (or none).
-   */
-  public static final String SERVER_SIDE_ENCRYPTION_AES256 =
```

Comment:
we can't pull values out of public interfaces like this, even if its not supported. They'd need to be tagged as deprecated